### PR TITLE
adds in basic coloring

### PR DESCRIPTION
**Pull request by bg451**
- adds in basic coloring, doesn't litter windows with escape characters
- some minor output formatting fixes

coloring

![Doesn't break windows!] (http://i.imgur.com/oC4wqHc.png)

### CODE REVIEW

**Comment by bg451** [comment]
@rdallman

**Comment by carimura** [comment]
colors and arrows!!!

File: main.go [hunk 1]
```diff
@@ -13,13 +16,15 @@ var (
 	// TODO(reed): kind of awkward, since there are 2 different flag sets now:
 	//  e.g.
 	//    ironcli -token=123456789 upload -max-concurrency=10 my_worker
-	versionFlag   = flag.Bool("version", false, "what year is it")
```
Comment (rdallman) [inline main.go]: wat

**Comment by rdallman** [comment]
sweet

**Pull request merged**